### PR TITLE
[remove]psql_docker script

### PR DESCRIPTION
**Pull request by k2khalsa**
created the release branch after merging feature with develop and hence had to use the master to compare

### CODE REVIEW

File: linux_sql/scripts/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#Setup arguments
+
+#Main code
+#check if docker_deamon is running - otherwise start run
+systemctl status docker || systemctl start docker
+
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:10]: please review the shell style guide and fix your code
https://www.notion.so/jarviscanada/Shell-Style-Guide-764a3ac0c9044fdb844b55420d1c7594

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  elif [ "$#" != 4 ]; then
+    echo "Error: database username or password not passed"
+    exit 1
+
+  else
+    docker pull postgres
+    docker volume create pgdata
+    export db_user=$2
+    export db_pass=$3
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: environment vars are not necessary here. please review my env video posted on slack today

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    exit $?
+    fi
+
+elif  [ "$(docker ps -a -f name=jrvs-psql | wc -l)" != "2" ]; then
+    echo "Error: Container not created"
+    exit 1
+
+else
+  if [ $1 == "start" ]; then
+    docker container start jrvs-psql
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:38]: don't you need to check the container existence before starting?

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    exit $?
+
+  elif [ $1 == "stop" ]; then
+    docker container start jrvs-psql
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: you do not need to start before stoping

**Review by jarviscanada — CHANGES_REQUESTED** [review]
fix all comments and request another review

File: linux_sql/scripts/psql_docker.sh (hunk 3 above)
Comment (k2khalsa) [reply linux_sql/scripts/psql_docker.sh:38]: Yes I do. I accidentally did it outside the else loop before. It is fixed now.

File: linux_sql/scripts/psql_docker.sh (hunk 4 above)
Comment (k2khalsa) [reply linux_sql/scripts/psql_docker.sh]: fixed

File: linux_sql/scripts/psql_docker.sh (hunk 2 above)
Comment (k2khalsa) [reply linux_sql/scripts/psql_docker.sh]: fixed

File: linux_sql/scripts/psql_docker.sh (hunk 1 above)
Comment (k2khalsa) [reply linux_sql/scripts/psql_docker.sh:10]: not sure about this, will check with edward tomorrow during office hours.

**Pull request closed without merging**